src/main/plugins/org.dita.base/xsl/preprocess/conrefImpl.xsl

```diff
@@ -547,7 +547,7 @@ See the accompanying LICENSE file for applicable license.
     <xsl:variable name="conref-topicid" as="xs:string">
       <xsl:choose>
         <xsl:when test="empty($topicid)">
-          <xsl:value-of select="//*[contains(@class, ' topic/topic ')][1]/@id"/>
+          <xsl:value-of select="/descendant-or-self::*[contains(@class, ' topic/topic ')][1]/@id"/>
```

Comment:
A classic XPath bug where `//` desugars to `/descendant-or-self::node()/` and the `[1]` predicate no longer selects the first match of `//`, but rather anything that is the first child of parent node.

Reply:
Ouch